packages/sdk/chainDeploy/mainnets/polygon.ts

```diff
@@ -437,6 +449,41 @@ const gelatoAssets: GelatoGUniAsset[] = [
   },
 ];
 
+const jarvisLiquidityPools: JarvisLiquidityPool[] = [
+  // jAUD -> USDC
+  { expirationTime: 40 * 60, liquidityPoolAddress: "0x36572797Cc569A74731E0738Ef56e3b8ce3F309c" },
```

Comment:
We have to ask the Jarvis team if we can use 0 for `expriationTime` , because in our use case it is used for a single atomic transaction (the liquidation) and we don't have the ability to wait for a later time for the mint/redeem to happen.